### ISSUE TITLE
Information about the error in input is not correctly announced by screen reader

### ISSUE THREAD

**Issue by Comandeer** [ckeditor/ckeditor5#1406]
## Is this a bug report or feature request? (choose one)

🐞 Bug report

## 💻 Version of CKEditor

https://ckeditor.com/docs/ckeditor5/latest/examples/builds/classic-editor.html

## 📋 Steps to reproduce

1. Launch screen reader.
2. Click "Insert media" button.
3. Type incorrect URL, e.g. "hublabubla" or just left the input empty.
4. Press <kbd>Enter</kbd>.

## ✅ Expected result

The screen reader reads the error message.

## ❎ Actual result

The screen reader only informs that field is invalid.

## 📃 Other details that might be useful

Tested in VoiceOver. Probably the issue is connected with the fact that element containing the error does not have appropriate `[aria-live]` or `[role]` attribute.
